src/ContactForm.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+
+class ContactForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      avatar: '',
+      occupation: '',
+    }
+  }
+
+  handleNameChange(event) {
+    this.setState({
+      name: event.target.value
+    });
+  }
+
+  handleOccupationChange(event) {
+    this.setState({
+      occupation: event.target.value
+    });
+  }
+
+  handleAvatarChange(event) {
+    this.setState({
+      avatar: event.target.value
+    });
+  }
```

Comment:
Did you see the extension in the curriculum about how to put the handling logic into one function?